From 50656e0d80441ddcdf8d259e713e237089503cd3 Mon Sep 17 00:00:00 2001
From: Govindraj Raja <Govindraj.Raja@imgtec.com>
Date: Thu, 29 Oct 2015 10:44:28 +0000
Subject: defconfig: enable usb persistent storage option

Keep USB persistent option enabled this is needed
to retain the state of root-fs on USB after coming
out of suspended state, this will re-mount root-fs
on usb while resuming the devices.

Change-Id: I1638d22f68c72eae863aa4e9206f76c803d9f400
Signed-off-by: Govindraj Raja <Govindraj.Raja@imgtec.com>
---
 arch/mips/configs/pistachio_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/mips/configs/pistachio_defconfig b/arch/mips/configs/pistachio_defconfig
index dc6ec4a..7938ddf 100644
--- a/arch/mips/configs/pistachio_defconfig
+++ b/arch/mips/configs/pistachio_defconfig
@@ -271,7 +271,6 @@ CONFIG_SND_SOC_IMG=y
 CONFIG_SND_SOC_IMG_PISTACHIO=y
 CONFIG_USB=y
 CONFIG_USB_ANNOUNCE_NEW_DEVICES=y
-# CONFIG_USB_DEFAULT_PERSIST is not set
 CONFIG_USB_OTG=y
 CONFIG_USB_MON=y
 CONFIG_USB_EHCI_HCD=y
-- 
1.9.1

